index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>To do list</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="container__title">
+            <h1>TO DO PLAN</h1>
+        </div>
+        <div class="container__content">
+            <div class="container__content-item">
```

Comment:
use just `item` instead of `container__content-item`, then change to children's name classes. Do the same for the other items